xonsh/timings.py

```diff
@@ -46,10 +46,10 @@ def clocku():
             """clocku() -> floating point number
             Return the *USER* CPU time in seconds since the start of the process.
             This is done via a call to resource.getrusage, so it avoids the
-            wraparound problems in time.clock()."""
+            wraparound problems in time.perf_counter()."""
```

Comment:
I think this is fine to keep as it is. The wraparound comment is not valid for `perf_counter()`, but still makes sense for clock().  You should either restore the comment to say clock() (which we are still avoiding) or remove the whole comment.  

The problem with clock is that it uses uint32s. So it was possible to reset times back to zero if you got to big.  `perf_counter()` uses floats under the covers and so doesn't have this issue.